internal/k8s/names.go

```diff
@@ -5,8 +5,14 @@ import (
 	"strings"
 )
 
+const FmtduplicateYamlDetectedError = "Duplicate YAML Entity(s): [%s] has been detected across one or more resources.  Only one specification per entity can be applied to the cluster; to ensure expected behavior, remove the duplicate specifications."
```

Comment:
Does this need to be exported? (It's exported because it begins with an uppercase letter)

Reply:
Yep! I'm using that `const` and function in `tiltfile_test.go` in `loadErrString()`

Reply:
hmm, I don't see `FmtduplicateYamlDetectedError` used anywhere but this file; this const can be private but accessed by a public function `DuplicateYamlDetectedError` elsewhere!

Small naming sidenote that doesn't super matter: I think you want `fmtDuplicate...` (capital D), and in general acronyms/initialisms like `YAML` should be [consistently cased](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) -- so this func should be `fmtDuplicateYAMLDetectedError`

Reply:
Ohhh right, I thought I wouldn't be able to access a private const, even through a public function. Will correct that and the casing!

Reply:
Corrected this, will merge into master now!